examples/wordpress/README.md

```diff
@@ -0,0 +1,145 @@
+# Demo: WordPress
+
+In this tutorial, you will learn - how to use `kustomize` to declare a variable reference and substitute it in container's command.
+
+In the WordPress example, we want 
```

Comment:
To run WordPress it's necessary to 

_in general avoid first person narrative, i.e. don't use "we"_